Makefile

```diff
@@ -1,15 +1,17 @@
 REPO=dealii
 
-RELEASES=v8.5.0
-DEPS=gcc-mpi-fulldepsmanual gcc-mpi-fulldepscandi clang-serial-bare
+RELEASES=9.0.0
```

Comment:
you need this without "v" because you want to do ``spack install dealii@VER``, right? Can we still name the label "v9.0.0..."?

Reply:
Yes, I'll do that